.github/workflows/build.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     if: github.ref == 'refs/heads/master' || github.event.pull_request.base.ref == 'master'
     runs-on: ubuntu-latest
     container:
-      image: fedora:35
+      image: fedora:latest
```

Comment:
@matejak This will pull the latest released Fedora container, not rawhide. Should we rather change this to `fedora:rawhide` image?

Reply:
This is probably OK, as it is about running tests, and they are not so tightly coupled with the Anaconda, and other things are more likely to cause problems in Rawhide.